src/utils/__tests__/internal.ts

```diff
@@ -853,9 +852,7 @@ describe('getIdentity', () => {
   let context: Context;
 
   beforeEach(() => {
-    context = dsMockUtils.getContextInstance({
-      getCurrentIdentity: entityMockUtils.getIdentityInstance(),
-    });
+    context = dsMockUtils.getContextInstance({});
```

Comment:
```suggestion
    context = dsMockUtils.getContextInstance();
```

Reply:
I tried that, and it gives different behavior. 

When passing `undefined` configureContext won't get called:

```
export function getContextInstance(opts?: ContextOptions): MockContext {
  if (opts) {
    configureContext({ ...defaultContextOptions, ...opts });
  }
  return mockInstanceContainer.contextInstance;
}
```

Reply:
You're missing the `beforeAll` with `dsMockUtils.initMocks()`